spec/acceptance/add_attribute_spec.rb

```diff
@@ -0,0 +1,33 @@
+describe "add_attribute" do
```

Comment:
A few thoughts about this test:

- We've been gradually moving away from using `let`, `before`, and `subject` for the reasons mentioned in this [blog post](https://thoughtbot.com/blog/lets-not). I'd prefer moving the setup into the test itself.
- I'd like to see some mention of assigning "[reserved words](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#method-name--reserved-word-attributes)" in the test description, since this is the main reason for using `#add_attribute`. 
-. I'm not sure we need separate tests for the `attributes_for` and `build` strategies here, since the strategies themselves are tested well enough elsewhere.

Putting that all together, I think we'd get something like:

```rb
describe "#add_attribute" do
  it "assigns attributes for reserved words" do
    define_model("Post", title: :string, sequence: :string, new: :boolean)
    
    FactoryBot.define do
      factory :post do
        add_attribute(:title) { "Title" }
        add_attribute(:sequence) { "Sequence" }
        add_attribute(:new) { true }
      end
    end

    post = FactoryBot.build(:post)

    expect(post.title).to eq "Title"   
    expect(post.sequence).to eq "Sequence"
    expect(post.new).to eq true
  end
end
```

Reply:
Hi, @composerinteralia! Thanks for the review, hope this will do =)